src/tabs/character.js

```diff
@@ -29,11 +30,11 @@ export const Character = ({open, game, wearActions, panelsRef, setOpen, toggleOp
             <div className={classnames(styles['panel-section'], styles['name-placeholder'])} />
             <div className={classnames(styles['panel-section'], styles['main-stats'])}>
               <div className={styles['panel-row']}>
-                <h2>HP</h2>
+                <h2 id = "healthDisplay">HP </h2>
```

Comment:
nit: The other attributes in this file do not having spaces, it is standard to not have spaces here.

```html
<h2 id="healthDisplay">HP </h2>
```